example/lib/test_scenarios/peripheral_test_operations.dart

```diff
@@ -113,14 +113,46 @@ class PeripheralTestOperations {
   }
 
   Future<void> writeCharacteristicForPeripheral() async {
-    log("Turning on temperature update");
+    log("Write to characteristic via peripheral");
     await peripheral.writeCharacteristic(
         SensorTagTemperatureUuids.temperatureService,
-        SensorTagTemperatureUuids.temperatureConfigCharacteristic,
-        Uint8List.fromList([1]),
+        SensorTagTemperatureUuids.temperatureDataCharacteristic,
+        Uint8List.fromList([0, 200, 40, 0]),
         false);
 
-    log("Turned on temperature update");
+    log("Written to characteristic");
+  }
+
+  Future<void> writeCharacteristicForService() async {
```

Comment:
IMHO it should be introduced in the develop and then merged to `blesimulator/develop`, if not then when will those changes appear in the develop?

Reply:
Done. https://github.com/Polidea/FlutterBleLib/pull/225